static/js/modules/game/articleRenderer.js

```diff
@@ -5,47 +5,44 @@ export class ArticleRenderer {
     /* frame: DOM element (i.e. through getElementById) to render article in
      * pageCallback: Called upon loading an article, should expect new page and load time
      */
-    constructor(frame, pageCallback, setupPreviews) {
+    constructor(frame, pageCallback, mouseEnter, mouseLeave) {
         this.frame = frame;
         this.pageCallback = pageCallback;
-        this.setupPreviews = setupPreviews;
+        this.mouseEnter = mouseEnter;
+        this.mouseLeave = mouseLeave;
     }
 
     async loadPage(page) {
+        try {
```

Comment:
I think many of the changes here is just a refactor to remove the extra loadPageWrapper we had before?